openapi/multi-yaml/index.yaml

```diff
@@ -59,6 +59,12 @@ tags:
       Payouts are processed each day at 11:30am US/Central time. A Platform can also configure each
       sub account to have an expedited payout priority. If this is enabled, the payout will be settled on the
       day the payout is generated. Otherwise, standard payouts will settle the next business day.
+  - name: Payout Holds
+    description: |
+      Payout Holds is a resource that allows platforms to temporarily hold or pause payouts for sub accounts.
```

Comment:
Can you change this to:
"A payout hold is a resource..."
Also, is it correct that a platform can hold or pause payouts?
I thought it can only be done by the JustiFi risk management team.